runtime/ftplugin/hare.vim

```diff
@@ -53,6 +50,11 @@ augroup hare.vim
   endif
 augroup END
 
+if !exists('current_compiler')
+  let b:undo_ftplugin ..= "| compiler make"
```

Comment:
```suggestion
  let b:undo_ftplugin .= "| compiler make"
```
to be consistent with the style a few lines above

Reply:
Will do. (A few lines further above it's `..` again.)